encoders/nlp/CLIPTextEncoder/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM jinaai/jina
+
+# install git
+RUN apt-get -y update && apt-get install -y git
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+# install the third-party requirements
+RUN pip install git+https://github.com/openai/CLIP.git
+
+# for testing the image
+RUN pip install pytest && pytest -s -vv
```

Comment:
Consider using multi-stage dockerfile, otherwise all test artifacts make it into the final image, bloating it considerably.
See `TransformerTorchEncoder` for an example

Reply:
you mean mergining the 2 RUNs into a single like this?
```
RUN pip install git+https://github.com/openai/CLIP.git && \
       pip install -r  /requirements.txt --no-cache-di
```

Reply:
No no, I meant doing like
``` docker
FROM jinaai/jina AS base
# build image here

FROM base
# test here - everything that is done/added here will be discarded and not part of the final image

FROM base
# From here on things are part of the final image again
ENTRYPOINT ["jina", "pod", "--uses", "config.yml"] 
```

Reply:
thanks for the comment, I will benchmark this in the future